ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss

```diff
@@ -140,10 +140,10 @@
 .govuk-grid-column-two-thirds__info_terms_use {
```

Comment:
Just with an eye to simplifying and reducing duplication; is there a reason these classes are so specific?

Reply:
@jacksonj04 yes it was based on design checks with Terry to make the cards align correctly as text amopunt etc and titles are different

Reply:
If we're doing something where each box needs different styles with some straightfoward text then there's probably something else we should be looking at, but that's a separate issue! 😄